doc/user/content/sql/create-source/_index.md

```diff
@@ -24,12 +24,13 @@ Materialize bundles **native connectors** that allow ingesting data from the fol
 {{< linkbox title="Message Brokers" >}}
 - [Kafka](/sql/create-source/kafka)
 - [Redpanda](/sql/create-source/kafka)
-- [Other message brokers](/integrations/#message-brokers)
 {{</ linkbox >}}
 {{< linkbox title="Databases (CDC)" >}}
-- [PostgreSQL](/sql/create-source/postgres)
-- [MySQL](/sql/create-source/mysql)
-- [Other databases](/integrations/#other-databases)
+- [PostgreSQL](/ingest-data/postgres/)
+- [MySQL](/ingest-data/mysql/)
+- [SQL Server](/ingest-data/cdc-sql-server/)
+- [MongoDB](https://github.com/MaterializeIncLabs/materialize-mongodb-debezium)
```

Comment:
Copied this list from  ingest_data page  -- we listed MongoDB there ....
Not sure what the status of the MongoDB stuff is --  There was discussion of both incorporating into docs as well as putting the mdb work on hold 😄